README.md

```diff
@@ -0,0 +1,89 @@
+<h1 style="text-align:center;">Pick Your Path</h1>
+
+
+__IDEA:__ 
+
+The father takes her daughter out for a late night walk, during the walk there is heavy wind and a faint whisper. Later some sort of creature snatches the girl and runs. The father sets off in the woods to find the __*monster*__, and get her daughter
+
```

Comment:
You need a section on here that shows the user how to actually run the game via the CLI.